rules/linux/credential_access_credential_dumping.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2023/02/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/02/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of the Linux utility 'unshadow' being executed with the right set of parameters.
+This technique could be used by malicious actors to retrieve the contents of the '/etc/shadow' and
+'/etc/password' files.Using these files the malicious threat actors can prepare
+themselves for password-cracking utilities.
```

Comment:
```suggestion
Identifies the use of the unshadow utility, which is part of the John the Ripper, a password-cracking tool. Attackers can use the
utility to combine the '/etc/passwd' and '/etc/shadow' files for input on password-cracking tools.
```

Reply:
unshadow is a vanilla linux utility ( its a  standalone command in linux). The command is available outside of John the Ripper. 
Will the actors use the combined file on password cracking or use it for Reconnaissance attacks for later purposes cannot be definitely defined. 
This description change mostly suggests one mode of attack or one mode of usage for unshadow purposes. 

While the initial description depicts the most braoduse case of password cracking. I will add the Reconnaissance part to the descript with some adverb changes, but the suggestion is too restricted and I am rephrasing it out of the suggestion.

Reply:
@shashank-elastic Is it? As far as I know unshadow is part of the john package. As a test I ran:

<img width="514" alt="image" src="https://user-images.githubusercontent.com/78494512/221853308-55535c54-146f-4f46-8a35-573d2ef43763.png">

And it does not seem to be a core unix utility unless the john package was installed.

Reply:
Yeah, AFAIK, it is part of John's package. I think recon is an edge case here. We should focus on credential access.

Reply:
https://manpages.ubuntu.com/manpages/xenial/man8/unshadow.8.html

Reply:
https://manpages.debian.org/jessie/john/unshadow.8.en.html

Reply:
I have a system full of exploits and packages! hence may be evaluated to be a vanilla linux command! 
I would still recommend the  Reconnaissance part, making aware of edge cases and possible scenarios to the end user is a critical part of the threat-hunting description. Will alter the command with package distribution!